src/Broadway/Domain/DateTime.php

```diff
@@ -26,6 +26,7 @@ class DateTime
 
     private function __construct(BaseDateTime $dateTime)
     {
+        $dateTime->setTimezone(new DateTimeZone('UTC'));
```

Comment:
We should at least mention this in the changelog.

We could also only force a copy to UTC when folks go comparing?